src/modules/mim/packagemanagerconfiguration.json

```diff
@@ -86,20 +86,28 @@
                       "enumValue": 3
                     },
                     {
-                      "name": "deserializingSources",
+                      "name": "deserializingGpgKeys",
                       "enumValue": 4
                     },
                     {
-                      "name": "modifyingSources",
+                      "name": "deserializingSources",
                       "enumValue": 5
                     },
                     {
-                      "name": "updatingPackagesLists",
+                      "name": "downloadingGpgKeys",
                       "enumValue": 6
                     },
                     {
-                      "name": "installingPackages",
+                      "name": "modifyingSources",
                       "enumValue": 7
+                    },
+                    {
+                      "name": "updatingPackagesLists",
```

Comment:
>updatingPackagesLists [](http://example.com/codeflow?start=31&length=21)

updatingPackageLists? ("Package" instead of "Packages")

Reply:
Renamed - please prioritize the merging of https://github.com/Azure/azure-osconfig/pull/169 before merging back this PR. We really need PR-169 to get merged back to make any further progress.

Reply:
Sounds good